gspread/utils.py

```diff
@@ -52,6 +52,15 @@ class Dimension(StrEnum):
     cols = "COLUMNS"
 
 
+class MergeType(StrEnum):
+    merge_all = "MERGE_ALL"
+    merge_columns = "MERGE_COLUMNS"
+    merge_rows = "MERGE_ROWS"
+
+
+SILENCE_WARNINGS_ENV_KEY = "GSPREAD_SILENCE_WARNINGS"
```

Comment:
is this used in v6?

Reply:
No we don't. But I thought we would keep it, for the next major release, this way we can use the exact same env variable